<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>343</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>624</width>
    <height>343</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Industriebaurichtlinie 2015</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Dateineingabe:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="verticalGroupBox_3">
          <property name="title">
           <string>Auswertung</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPushButton" name="pushStart">
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="verticalGroupBox">
          <property name="title">
           <string>Informationen</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="pushAnw">
             <property name="text">
              <string>Anwendungsbereich</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushLin">
             <property name="text">
              <string>Links</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Vorgangsnummer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="fileEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Frimenname</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Straße und Hausnummer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="streetEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>PLZ und Stadt</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="townEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Quadratmeter je BBK in m²</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="squareSpinBox">
          <property name="maximum">
           <double>999999.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>lichte Höhe in m</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="heightSpinBox"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Anzahl der Obergeschosse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="topSpinBox">
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Anzahl der Untergeschosse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="subSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Feuerwiderstandsklasse</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="baustoffCombo">
            <item>
             <property name="text">
              <string>F0-A</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F60-A</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F90-A</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Sicherheitskategorie</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sicherheitsCombo">
            <item>
             <property name="text">
              <string>K1 Keine</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>K2 BMA</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>K3.1 Staffel</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>K3.2 Gruppe</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>K3.3 2 Staffel</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>K3.4 3 Staffel</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>K4 Sprinkler</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushNeu">
        <property name="text">
         <string>Neuladen</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushBeenden">
        <property name="text">
         <string>Beenden</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>label_7</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>label_2</zorder>
   <zorder>verticalSpacer</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>642</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Datei</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
